src/backend/base/langflow/services/cache/service.py

```diff
@@ -245,6 +247,10 @@ async def set(self, key, value, lock=None) -> None:
             msg = "RedisCache only accepts values that can be pickled. "
             raise TypeError(msg) from exc
 
+    @override
+    async def set(self, key, value, lock=None) -> None:
+        self._set(key, value)
```

Comment:
Calling blocking method here 🤔.
Should use the `redis.asyncio` client ?

Reply:
Sure